gluoncv/logs/classification/imagenet/vgg16.sh

```diff
@@ -0,0 +1 @@
+python train_imagenet.py --batch-size 32 --num-gpus 8 -j 64 --mode hybrid --num-epochs 100 --lr 0.01 --lr-decay 0.1 --lr-decay-epoch 50,80 --log-interval 1000 --model vgg16
```

Comment:
Please remove `--log-interval 1000` as I don't have them in my training, thus it will not match the log we have.

Reply:
done